### PR TITLE
[jellyfish-merkle] Test consecutive contiguous account updates

### PR DESCRIPTION
**Pull request by zamsden**
Testing a modified Jellyfish tree revealed some strange behavior
in this edge case.  Adding a test to Libra to make sure we have
coverage.

## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: storage/jellyfish-merkle/src/jellyfish_merkle_test.rs [hunk 1]
```diff
@@ -637,6 +637,44 @@ proptest! {
         test_existent_keys_impl(&tree, version, &kvs);
     }
 
+    #[test]
+    fn test_put_blob_with_proof2(
```
Comment (msmouse) [inline storage/jellyfish-merkle/src/jellyfish_merkle_test.rs]: why calling it xxx2? (because there seems to be no xxx1)

Comment (zamsden) [reply storage/jellyfish-merkle/src/jellyfish_merkle_test.rs]: Because it tests 2 accounts

Comment (wqfish) [reply storage/jellyfish-merkle/src/jellyfish_merkle_test.rs]: I don't feel this test is a lot different from the one above except that it adds a few assertions. Maybe just add these assertions to `init_mock_db` so every test gets them?

**Comment by davidiw** [comment]
Closing this out as its been sitting here for a looong time. Feel free to re-open.
